docs/workspace/meas/index.md

```diff
@@ -1,2 +1,80 @@
 # Measurement Electronics
+## Research on components
 
+ **Current Measurement:** A current sense amplifier or a shunt resistor in combination with an ADC can be used for 
+ current measurement. The shunt resistor is placed in the path of the current to be measured, and the voltage 
+ drop across it (which is proportional to the current) is measured.
+
+**Voltage Measurement:** A high-precision ADC (Analog-to-Digital Converter) can be used for voltage measurement.
+ It can convert the analog voltage signal into a digital value for processing. A suitable ADC would be one with 
+ a resolution high enough to achieve your desired accuracy of 1mV in a 0-5V range.
+
+
+### Choice for Current measurement:
+
+ - **Hall Effect Sensors:** These devices measure current and voltage through magnetic fields and provide galvanic 
+ isolation. They are suitable for both AC and DC measurements.
+
+**Available option:**
+
+
+- ACS712-20A/ ACS714-20A:
+
+
+[ACS712-20A/ ACS714-20A datasheet]
+
+- TMCS1100:
+
+
+[TMCS1100 datasheet]
+
+The ACS712-20A and the ACS714-20A both serve the same purpose, but the ACS714-20A is better suited for harsher 
+environments. Considering the controlled testing environment and cost factors, I recommend opting for the **ACS712-20A** in this case. 
+
+### Choice of FUSE:
+
+- Thermal Fuse or Switch: This is used to protect the circuit from excessive currents. It should be connected between 
+I_MEAS+ and I_MEAS- and should be able to disconnect the circuit within a specified time if the current exceeds a certain limit.
+
+**Available option:**
+
+  1.	BOURNS CB72ABB: This is a thermal fuse with a functioning temperature of 72°C and a holding temperature of 40°C.
+
+  2.	BOURNS CB85ABB: This thermal fuse has a functioning temperature of 85°C and a holding temperature of 40°C.
+
+  3.	BOURNS SC82AAA: This thermal fuse has a functioning temperature of 82°C and a holding temperature of 40°C.
+
+  4.	BOURNS CB77ABB: This thermal fuse has a functioning temperature of 77°C and a holding temperature of 40°C.
+
+  5.	BOURNS SC85AAB: This thermal fuse has a functioning temperature of 85°C and a holding temperature of 40°C
+
+
+ [Datasheet for fuses]
+
+I would like to go with the below two options because there is ***no need to replace*** them after each time the fuse goes off,
+ unlike thermal fuses or switches.
+
+- **Electronic Fuses (eFuses):** These active devices use a sense resistor and a FET to monitor current flow. When the 
+current exceeds a preset value, the control logic turns the FET off, cutting the flow of current.
+
+Electronic fuses are not suitable for the current in use, as they are typically designed for very small currents. 
+Therefore, circuit breakers would be a better choice
+
+- **Circuit Breakers:** These are resettable switches that open automatically when the current exceeds a certain threshold. 
+They can be manual or automatic and are suitable for repeated use.
```

Comment:
I think the use of a "resettable" fuse, which is not thermally destroyed is beneficial.

- What would be the order of magnitude for the maximum current of eFuses?
- Circuit breakers are an excellent idea! I think this would be very interesting for the emergency stop unit team as a replacement for a thermal fuse aswell. (@mohamedshafi07 @Ranjithraj2001 @KovarthananS) It would also yield the benefit, that a circuit breaker could be bought which is officially "safety certified".
- From my feeling a circuitbreaker is a little bit "overkill" for the measurement component. This overcurrent protection does not need to protect lives or prevent great damage like fire. It should just limit damage to the measurement device. After a research online I found that there are [resettable fuses](https://en.wikipedia.org/wiki/Resettable_fuse) which are called _polymeric positive temperature coefficient device (PPTC)_. This also reminds me of genral _positive temperature coefficient devices_ (PTC) which should show similar behavior.
@ruchik-mahesh do you think that this could be a simpler and cheaper alternative for a circuitbreaker?

Reply:
About the eFuses the max is 10A but again the normal efuses that are used go up to 5A, So i did not consider it, but you did tell a way around for this, but i dont think taking all that trouble would be required, if we use a thermal fuse as you stated.